tests/manage/monitoring/conftest.py

```diff
@@ -712,3 +712,51 @@ def do_nothing():
     test_file = os.path.join(measurement_dir, 'measure_workload_idle.json')
     measured_op = measure_operation(do_nothing, test_file)
     return measured_op
+
+
+@pytest.fixture
+def measure_stop_rgw(measurement_dir, request, rgw_deployments):
+    """
+    Downscales RGW deployments, measures the time when it was
+    downscaled and monitors alerts that were triggered during this event.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for stopping
+            RGW pods
+
+    """
+    oc = ocp.OCP(
+        kind=constants.DEPLOYMENT,
+        namespace=config.ENV_DATA['cluster_namespace']
+    )
+
+    def stop_rgw():
+        """
+        Downscale RGW interface deployments for 5 minutes.
+
+        Returns:
+            str: Name of downscaled deployment
+
+        """
+        # run_time of operation
+        run_time = 60 * 5
+        nonlocal oc
+        nonlocal rgw_deployments
+        for rgw_deployment in rgw_deployments:
+            rgw = rgw_deployment['metadata']['name']
+            logger.info(f"Downscaling deployment {rgw} to 0")
+            oc.exec_oc_cmd(f"scale --replicas=0 deployment/{rgw}")
+        logger.info(f"Waiting for {run_time} seconds")
+        time.sleep(run_time)
```

Comment:
how about adding logging to happen every few seconds, so there will be some indication of what's going on with the RGW deployment?

Reply:
During this time is running other [thread which logs coming alerts](https://github.com/red-hat-storage/ocs-ci/blob/de5d0abb024121dc82c28be26ce741ba94fb2f2b/ocs_ci/utility/workloadfixture.py#L65) and prints them every few seconds.